ci-operator/config/coreos/coreos-assembler/coreos-coreos-assembler-master.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    namespace: coreos
+    name: cosa-buildroot
+    tag: latest
```

Comment:
Note this image is still not being triggered correctly. I had to manually start a build recently. I thought that https://github.com/openshift/release/pull/5488#issuecomment-557110560 would have fixed it, but I'm guessing we're missing something else here.

Reply:
Yeah :cry:

Reply:
I think this might be fixed now. Somehow the imagestream `coreos-assembler` didn't have `scheduled: true` even though the YAML specifies it. I fixed that now and it imported the latest cosa, which in turn triggered a new build of `cosa-buildroot`. We'll see if that field gets nuked again on the next refresh.